pkg/services/services.go

```diff
@@ -0,0 +1,22 @@
+package services
+
+import (
+	"github.com/openshift/windows-machine-config-operator/pkg/servicescm"
+	"github.com/openshift/windows-machine-config-operator/pkg/windows"
+)
+
+// GenerateServicesManifest returns the expected state of the Windows service configmap
+func GenerateServicesManifest() (*servicescm.Data, error) {
```

Comment:
I suggest making this `GenerateManifest()` so that the call site does not stutter.